### PR TITLE
feature 1510 v4.1.0 Release

### LINKED ISSUE #1510: Create the METplus v4.1.0 release

### PR DESCRIPTION
**Pull request by georgemccabe**
Review release notes: https://metplus.readthedocs.io/en/feature_1510_v4.1.0/Users_Guide/release-notes.html#metplus-version-4-1-0-release-notes-2022-03-14

### CODE REVIEW

File: docs/Users_Guide/release-notes.rst [hunk 1]
```diff
@@ -38,198 +38,193 @@ When applicable, release notes are followed by the GitHub issue number which
...
+    * Add option to TCMPRPlotter to pass in directory to tc_stat instead of individual files (`#1057 <https://github.com/dtcenter/METplus/issues/1057>`_)
+    * Add option to pass in the input directory to TCMPRPlotter instead of finding all tcst files and passing the list (`#1084 <https://github.com/dtcenter/METplus/issues/1084>`_)
+
+  * CyclonePlotter:
+
+    * Update CyclonePlotter for offline/HPC usage (`#933 <https://github.com/dtcenter/METplus/issues/933>`_)
+    * CyclonePlotter, create options to format output grid area to user-desired area (`#1091 <https://github.com/dtcenter/METplus/issues/1091>`_)
+    * CyclonePlotter, connected lines run over the Prime Meridian (`#1000 <https://github.com/dtcenter/METplus/issues/1000>`_)
+
+  * Use Cases:
```
Comment (j-opatz) [inline docs/Users_Guide/release-notes.rst:138]: The bullets following this header are at the same indentation level as the header, need to indent them in one.

**Comment by j-opatz** [comment]
I'm currently a bit lost as to the purpose of listing some entries as bold and not others. Is it done purely on suspected user interest, or certain funding support? Maybe the impact it has on the METplus suite? If it's subjective, as in "EMC was really looking forward to this so let's call it out", I'd suggest we find a more objective standard (e.g. number of coding lines changed, [n] new variables created, >[n] hours of work put into the issue, etc.). 
My reasoning is the work that's been done for CPC on HSS, GenEnsProd, BRIERCL, etc. is incredibly important to them and I'm sure they'd love a bold entry for their efforts, but EMC or the Met Office wouldn't have identical priorities when searching through release notes. If a subjective system is established for bold entries (or none are), the rationale can be clearly explained and adjusted as needed. No mysteries/FOMOs.

File: docs/Users_Guide/release-notes.rst (hunk 1 above)
Comment (georgemccabe) [reply docs/Users_Guide/release-notes.rst:138]: I made this change, thanks for pointing it out.

**Comment by JohnHalleyGotway** [comment]
@j-opatz I certainly don't have any clear direction on the bolding. It is a very subjective way of indicating emphasis.

Sometimes it's easy to pick out more impactful vs less impactful changes. For example, in the Gen-Vx-Mask [MET-10.1.0 release notes](https://met.readthedocs.io/en/develop/Users_Guide/overview.html#met-version-10-1-0-release-notes-20220314):

- **Refine logic to prevent rounding shapefile points to the nearest grid point** (affects GenVxMask -type shape masks) ([#1810](https://github.com/dtcenter/MET/issues/1810)).

Is bold because that change will likely modify existing outputs. Whereas:

- Change -type for Gen-Vx-Mask from an optional argument to a required one ([#1792](http://github.com/dtcenter/MET/issues/1792)).

Is not bold because its pretty minor. Although it does change the existing usage statement, users will now see an error message if they fail to provide it. And It doesn't change the output.

This could be a good topic for discussion at the next release retrospective. What standard should be applied when deciding whether or not to bold a release note?

Although applying OpenMP to Grid-Stat does NOT change outputs, but is listed in bold because it significantly impacts the performance.

I think choosing a single standard for **significant changes** to be emphasized that applies well in all cases is going to be difficult to impossible.

**Review by j-opatz — APPROVED** [review]
The larger discussion of what to bold/what not to bold can be saved for a retrospective and the METplus group at large. 

The notes as they currently are is sufficient for release, and provide a sensible flow between tools, general enhancements, bug fixes, etc.

**Pull request merged**